data/421/172/075/421172075.geojson

```diff
@@ -10,13 +10,491 @@
     "geom:latitude":24.157517,
     "geom:longitude":54.597557,
     "iso:country":"AE",
+    "label:ara_x_preferred_longname":[
+        "\u0623\u0628\u0648\u0638\u0628\u064a"
```

Comment:
I think you're missing a space between the words?

Reply:
fixed in most recent commit.